PRIVACY-POLICY.md

```diff
@@ -0,0 +1,20 @@
+<h1 align=center> PRIVACY POLICY </h1>
+This privacy policy describes what, how, and why <em>AntAlmanac</em> and the technologies that it implements collect, store, and utilize the information that you or your device shares through your use of the website. We take reasonable precautions to protect your data, but you should NOT enter sensitive information into <em>AntAlmanac</em>. We may change this policy when necessary. By using <em>AntAlmanac</em>, you consent to this privacy statement.
+
+<h2> Information that we collect </h2>
+<b> Non-tracking information: </b> 
+Information stored on your device that we do not log and that is not used to track usage. This includes, but is not limited to, settings, the last profile loaded on a device, and whether you have dismissed certain popups. This information allows <em>AntAlmanac</em> to make your actions and preferences persistent across page accesses.
+<br></br>
+<b> Study plans and usernames: </b> 
+Your username(s) and the associated schedules and enrolled classes. This allows you to save your schedules and is NOT shared with third parties.
+<br></br>
+<b> Google Analytics usage data: </b> 
+We use <em>Google Analytics</em> to track your usage, for example, which button you click on our your search queries. We do so to inform our prioritization and design of features. <em>Google Analytics</em> uses cookies and other tracking technologies to collect and store your information. Your usage information is anonymous and aggregated to us. HOWEVER, <em>Google Analytics</em> might collect your information individually or track you across websites. To block <em>Google Analytics</em>, use <em>Google</em>’s first-party <a href="https://support.google.com/analytics/answer/181881?hl=en">browser add-on</a>.
+<br></br>
+<em>
+  We do not collect any personally identifiable information. Your username should not contain your real name, and you should not enter sensitive information into AntAlmanac.
```

Comment:
Since we had a "information we collect" section, would it make sense to have a "information we *don't* collect section"?

Reply:
~~There is no information we don't collect 😈~~
JK, this is a good idea.